client/src/components/SignUpForm/SignUpForm.module.scss

```diff
@@ -0,0 +1,94 @@
+.formBody {
```

Comment:
Please check guidelines about naming style classnames

https://sass-guidelin.es/#naming-conventions

Sass recommends to use kebab case, so it is better to  name it `form-body `